core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -28,6 +28,7 @@ import kafka.utils.Implicits._
 import kafka.utils._
 import kafka.zk.{AdminZkClient, KafkaZkClient}
 import org.apache.kafka.clients.CommonClientConfigs
+import org.apache.kafka.clients.{admin => jadmin}
```

Comment:
nit: We only use `jadmin` for `jadmin.Config`. Maybe we ca do the same thing we do for `AdminClient` and rename the `Config` import. E.g. `import org.apache.kafka.clients.admin.{Config => JConfig}`.

Reply:
Yeah, makes sense. Thanks for the suggestion.